bris-adapt/pyproject.toml

```diff
@@ -37,8 +37,11 @@ dependencies = [
     "metpy>=1.7.1",
     "twine>=6.2.0",
     "torch-geometric==2.4.0",
+    "cartopy>=0.21.1",
+    "gridpp",
```

Comment:
This does not work on a mac

Reply:
Either make it an optional dependency or replace with scipy

Reply:
replaced